.github/workflows/devcontainer-feature-release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release dev container feature image
+
+env:
+  IMAGE_NAME: ${{ github.repository }}
+  REGISTRY: ghcr.io
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "ext/devcontainer/**"
+      - ".github/workflows/devcontainer-feature-release.yml"
+    branches: [main]
```

Comment:
PR triggers aren't needed for release. They're properly covered in the test workflow below :)